<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeOrderForm</class>
 <widget class="QDialog" name="TypeOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать услугу</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelServiceName">
     <property name="text">
      <string>Название услуги:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServiceName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelMaterial">
     <property name="text">
      <string>Материал:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxMaterial"/>
   </item>
   <item>
    <widget class="QLabel" name="labelQuantity">
     <property name="text">
      <string>Количество:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxQuantity">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
